src/DynamoCoreWpf/Controls/Login.xaml

```diff
@@ -93,6 +94,17 @@
                 </Style>
 
             </Button.Style>
+            <Button.ToolTip>
+                <StackPanel>
+                    <TextBox FontWeight="Bold" BorderThickness="0"
+                             Text="{x:Static p:Resources.Autodesk360SignInButtonTitleToolTip }"
+                             Background="Transparent"/>
+                    <TextBox Text="{x:Static p:Resources.Autodesk360SignInButtonContentToolTip}"
+                             Width="300"
+                             BorderThickness="0" TextWrapping="Wrap"
+                             Background="Transparent"/>
+                </StackPanel>
+            </Button.ToolTip>
```

Comment:
There's a space after `Autodesk360SignInButtonTitleToolTip`. Also, can you perform a `CTRL+K+D` to format the whole XAML file? Thanks!